app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
It's a good practice to keep constants in uppercase. Please, rename 'URL' to 'API_URL' for better understanding and to follow the code style.